internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>splunk-otel-collector</id>
+    <version>0.0.0</version>
+    <description>
+The splunk-otel-collector is a metric-based agent written in Go that is used to monitor infrastructure and application services from a variety of environments.
+
+The splunk-otel-collector looks for a configuration file at `\ProgramData\Splunk\OpenTelemetry Collector\otelcol.exe`.  If this file does not already exist during installation, a default config file will be copied into place by the installer.
+
+#### Package Parameters
+The following package parameters are available:
+

```

Comment:
We'll need to add a `MODE` option for `agent` or `gateway`, and should default to `agent`.  The `SPLUNK_CONFIG` registry value should then be configured for the config path based on this value, either `\ProgramData\Splunk\OpenTelemetry Collector\agent_config.yaml` or `\ProgramData\Splunk\OpenTelemetry Collector\gateway_config.yaml`.  The config should be copied from `\Program Files\Splunk\OpenTelemetry Collector` if it does not already exist in `\ProgramData\Splunk\OpenTelemetry Collector`.

Reply:
`MODE` option is added. so accordingly `SPLUNK_CONFIG` value will be set.